conf/messages

```diff
@@ -75,15 +75,16 @@ moreThanTwoHalfMill.heading = Was the estate worth more than £2.5 million at th
 moreThanTwoHalfMill.checkYourAnswersLabel = Was the estate worth more than £2.5 million at the date of death?
 moreThanTwoHalfMill.error.required = Select yes if the estate was worth more than £2.5 million at the date of death
 
-doNotNeedToRegister.title = Based on your answers, you do not need to register this estate
-doNotNeedToRegister.heading = Based on your answers, you do not need to register this estate
-doNotNeedToRegister.checkYourAnswersLabel = Based on your answers, you do not need to register this estate
-doNotNeedToRegister.p1 = You will need to use the
-doNotNeedToRegister.payment.link = informal payment arrangements (opens in a new tab or window)
-doNotNeedToRegister.p2 = if:
-doNotNeedToRegister.b1 = less than £10,000 of tax is due from income other than savings
-doNotNeedToRegister.b2 = tax due from savings exceeds £100 in any one tax year
-doNotNeedToRegister.p3 = If you need help with the service contact
+doNotNeedToRegister.title = You do not need to register this estate
+doNotNeedToRegister.heading = You do not need to register this estate
+doNotNeedToRegister.p1 = Your estate is a ‘simple estate’, you can tell HMRC about tax by using the
```

Comment:
This ties off with the spreadsheet they provided but not the screenshot which says "If the estate is a..." not "your estate is a..."

Reply:
Im not sure which version we should be using will ask Design team and post here when I hear back.

Reply:
Design team confirmed the csv is correct.